src/index.ts

```diff
@@ -35,14 +35,10 @@ export async function install(packageName: string, hideUserPrompt = false): Prom
     dependencies.map(async (item) => {
       if (Array.isArray(item)) {
         return Promise.all(
-          item.map(async (subitem) => ({
-            ...subitem,
-            directory: await resolveDependencyPath(subitem.name),
-          })),
+          item.map(async (subitem) => getResolvedDependency(subitem)),
```

Comment:
```suggestion
          item.map(getResolvedDependency),
```

Reply:
Thanks for the catch. I actually opened an Eslint issue about this myself. 😄 That's why I need tools to remind me of those.

https://github.com/eslint/eslint/issues/14065